test/features/authentication_test.rb

```diff
@@ -0,0 +1,24 @@
+require './test/test_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.